### PR TITLE
 ddl: using pingcap/log and unify the log format

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve? 
Using pingcap/log to replace logru in ddl pkg.

### Check List 

Tests

### CODE REVIEW

**Comment by crazycs520** [comment]
@zimulala @winkyao PTAL

File: ddl/column.go [hunk 1]
```diff
@@ -177,7 +178,7 @@ func onAddColumn(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, err error)
 			job.State = model.JobStateCancelled
 			return ver, errors.Trace(err)
 		}
-		log.Infof("[ddl] add column, run DDL job %s, column info %#v, offset %d", job, columnInfo, offset)
+		log.Info("run add column job", zap.String("job", job.String()), zap.Any("column", *columnInfo), zap.Int("offset", offset))
```
Comment (winkyao) [inline ddl/column.go]: column --> columnInfo

File: ddl/ddl.go [hunk 2]
```diff
@@ -291,7 +292,7 @@ type ddlCtx struct {
 
 func (dc *ddlCtx) isOwner() bool {
 	isOwner := dc.ownerManager.IsOwner()
-	log.Debugf("[ddl] it's the DDL owner %v, self ID %s", isOwner, dc.uuid)
+	log.Debug("DDL check is owner", zap.Bool("isOwner", isOwner), zap.String("selfID", dc.uuid))
```
Comment (winkyao) [inline ddl/ddl.go]: DDL check whether is the owner

File: ddl/ddl_worker.go [hunk 3]
```diff
@@ -538,20 +539,20 @@ func (w *worker) runDDLJob(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64,
 	if err != nil {
 		// If job is not cancelled, we should log this error.
 		if job.State != model.JobStateCancelled {
-			log.Errorf("[ddl-%s] run DDL job err %v", w, errors.ErrorStack(err))
+			log.Error("DDL worker run job error", zap.String("worker", w.String()), zap.Error(err))
 		} else {
-			log.Infof("[ddl-%s] the DDL job is normal to cancel because %v", w, err)
+			log.Info("DDL job is normal cancelled", zap.String("worker", w.String()), zap.Error(err))
```
Comment (winkyao) [inline ddl/ddl_worker.go]: normally

File: ddl/reorg.go [hunk 4]
```diff
@@ -191,6 +195,14 @@ type reorgInfo struct {
 	PhysicalTableID int64
 }
 
+func (r *reorgInfo) String() string {
+	return "StartHandle:" + strconv.FormatInt(r.StartHandle, 10) + "," +
```
Comment (winkyao) [inline ddl/reorg.go]: Why not use `strconv.Itoa`

File: ddl/ddl.go [hunk 5]
```diff
@@ -320,7 +321,7 @@ func asyncNotifyEvent(d *ddlCtx, e *util.Event) {
 			case d.ddlEventCh <- e:
 				return
 			default:
-				log.Warnf("Fail to notify event %s.", e)
+				log.Warn("fail to notify ddl event", zap.String("event", e.String()))
```
Comment (zimulala) [inline ddl/ddl.go]: s/ddl/DDL

File: ddl/reorg.go (hunk 4 above)
Comment (crazycs520) [reply ddl/reorg.go]: `StartHandle` is `int64`, `strconv.Itoa` need type `int`. 
Actually `strconv.Itoa` also use `strconv.FormatInt`.

File: ddl/ddl.go (hunk 5 above)
Comment (crazycs520) [reply ddl/ddl.go]: done.

File: ddl/reorg.go [hunk 6]
```diff
@@ -133,7 +136,7 @@ func (w *worker) runReorgJob(t *meta.Meta, reorgInfo *reorgInfo, lease time.Dura
 	select {
 	case err := <-w.reorgCtx.doneCh:
 		rowCount, _ := w.reorgCtx.getRowCountAndHandle()
-		log.Infof("[ddl] run reorg job done, handled %d rows", rowCount)
+		log.Info("[ddl] run reorg job done", zap.Int64("backRowCount", rowCount))
```
[the remark this answers is not in the document]
Comment (crazycs520) [reply ddl/reorg.go]: done.

File: ddl/index.go [hunk 7]
```diff
@@ -1224,7 +1230,7 @@ func (w *worker) updateReorgInfo(t table.PartitionedTable, reorg *reorgInfo) (bo
 	pid, err := findNextPartitionID(reorg.PhysicalTableID, pi.Definitions)
 	if err != nil {
 		// Fatal error, should not run here.
-		log.Errorf("[ddl-reorg] update reorg fail, %v error stack: %s", t, errors.ErrorStack(err))
+		log.Error("[ddl] find next partition id failed", zap.String("table", fmt.Sprintf("%v", t)), zap.Error(err))
```
[the remark this answers is not in the document]
Comment (crazycs520) [reply ddl/index.go]: done.

File: ddl/ddl_worker.go [hunk 8]
```diff
@@ -445,7 +445,7 @@ func (w *worker) waitDependencyJobFinished(job *model.Job, cnt *int) {
 	if job.DependencyID != noneDependencyJob {
 		intervalCnt := int(3 * time.Second / waitDependencyJobInterval)
 		if *cnt%intervalCnt == 0 {
-			log.Info("[ddl] DDL job need to wait dependent job, sleeps a while, then retries it.", zap.String("worker", w.String()), zap.Int64("jobID", job.ID), zap.Int64("dependentJobID", job.DependencyID), zap.Duration("waitTime", waitDependencyJobInterval))
+			logutil.Logger(context.Background()).Info("[ddl] DDL job need to wait dependent job, sleeps a while, then retries it.", zap.String("worker", w.String()), zap.Int64("jobID", job.ID), zap.Int64("dependentJobID", job.DependencyID), zap.Duration("waitTime", waitDependencyJobInterval))
```
Comment (zimulala) [inline ddl/ddl_worker.go]:
```suggestion
			logutil.Logger(context.Background()).Info("[ddl] DDL job need to wait dependent job, sleeps a while, then retries it.", 
						zap.String("worker", w.String()), 
						zap.Int64("jobID", job.ID), 
						zap.Int64("dependentJobID", job.DependencyID), 
						zap.Duration("waitTime", waitDependencyJobInterval))

File: ddl/ddl_worker.go [hunk 9]
```diff
@@ -539,20 +539,20 @@ func (w *worker) runDDLJob(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64,
...
 		job.ErrorCount++
 		// Load global ddl variables.
 		if err1 := loadDDLVars(w); err1 != nil {
-			log.Error("[ddl] load DDL global variable failed", zap.String("worker", w.String()), zap.Error(err1))
+			logutil.Logger(context.Background()).Error("[ddl] load DDL global variable failed", zap.String("worker", w.String()), zap.Error(err1))
 		}
 		// Check error limit to avoid falling into an infinite loop.
 		if job.ErrorCount > variable.GetDDLErrorCountLimit() && job.State == model.JobStateRunning && admin.IsJobRollbackable(job) {
-			log.Warn("[ddl] DDL job error count exceed the limit, cancelling it now", zap.String("worker", w.String()), zap.Int64("jobID", job.ID), zap.Int64("errorCountLimit", variable.GetDDLErrorCountLimit()))
+			logutil.Logger(context.Background()).Warn("[ddl] DDL job error count exceed the limit, cancelling it now", zap.String("worker", w.String()), zap.Int64("jobID", job.ID), zap.Int64("errorCountLimit", variable.GetDDLErrorCountLimit()))
```
Comment (zimulala) [inline ddl/ddl_worker.go]: Ditto.

File: ddl/ddl_worker.go [hunk 10]
```diff
@@ -629,7 +629,7 @@ func (w *worker) waitSchemaChanged(ctx context.Context, d *ddlCtx, waitTime time
 			return
 		}
 	}
-	log.Info("[ddl] wait latest schema version changed", zap.String("worker", w.String()), zap.Int64("ver", latestSchemaVersion), zap.Duration("takeTime", time.Since(timeStart)), zap.String("job", job.String()))
+	logutil.Logger(context.Background()).Info("[ddl] wait latest schema version changed", zap.String("worker", w.String()), zap.Int64("ver", latestSchemaVersion), zap.Duration("takeTime", time.Since(timeStart)), zap.String("job", job.String()))
```
Comment (zimulala) [inline ddl/ddl_worker.go]: Ditto.

File: ddl/index.go [hunk 11]
```diff
@@ -1022,15 +1022,15 @@ func (w *worker) handleReorgTasks(reorgInfo *reorgInfo, totalAddedCount *int64,
 			return errors.Trace(reorgInfo.UpdateReorgMeta(txn, nextHandle, reorgInfo.EndHandle, reorgInfo.PhysicalTableID))
 		})
 		metrics.BatchAddIdxHistogram.WithLabelValues(metrics.LblError).Observe(elapsedTime.Seconds())
-		log.Warn("[ddl] add index worker handle batch tasks failed", zap.Int64("totalAddedCount", *totalAddedCount), zap.Int64("startHandle", startHandle), zap.Int64("nextHandle", nextHandle),
+		logutil.Logger(context.Background()).Warn("[ddl] add index worker handle batch tasks failed", zap.Int64("totalAddedCount", *totalAddedCount), zap.Int64("startHandle", startHandle), zap.Int64("nextHandle", nextHandle),
```
Comment (zimulala) [inline ddl/index.go]: Ditto.

File: ddl/index.go [hunk 12]
```diff
@@ -1247,7 +1247,7 @@ func (w *worker) updateReorgInfo(t table.PartitionedTable, reorg *reorgInfo) (bo
 	err = kv.RunInNewTxn(reorg.d.store, true, func(txn kv.Transaction) error {
 		return errors.Trace(reorg.UpdateReorgMeta(txn, reorg.StartHandle, reorg.EndHandle, reorg.PhysicalTableID))
 	})
-	log.Info("[ddl] job update reorgInfo", zap.Int64("jobID", reorg.Job.ID), zap.Int64("partitionTableID", pid), zap.Int64("startHandle", start), zap.Int64("endHandle", end), zap.Error(err))
+	logutil.Logger(context.Background()).Info("[ddl] job update reorgInfo", zap.Int64("jobID", reorg.Job.ID), zap.Int64("partitionTableID", pid), zap.Int64("startHandle", start), zap.Int64("endHandle", end), zap.Error(err))
```
Comment (zimulala) [inline ddl/index.go]: Ditto.

File: ddl/syncer.go [hunk 13]
```diff
@@ -359,13 +359,13 @@ func (s *schemaVersionSyncer) OwnerCheckAllVersions(ctx context.Context, latestV
 
 			ver, err := strconv.Atoi(string(kv.Value))
 			if err != nil {
-				log.Info("[ddl] syncer check all versions, convert value to int failed, continue checking.", zap.String("ddl", string(kv.Key)), zap.String("value", string(kv.Value)), zap.Error(err))
+				logutil.Logger(context.Background()).Info("[ddl] syncer check all versions, convert value to int failed, continue checking.", zap.String("ddl", string(kv.Key)), zap.String("value", string(kv.Value)), zap.Error(err))
```
Comment (zimulala) [inline ddl/syncer.go]: Ditto.

**Pull request merged**